Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.h

```diff
@@ -48,6 +49,7 @@ namespace AzToolsFramework::Prefab
         // PrefabFocusInterface overrides ...
         void InitializeEditorInterfaces() override;
         PrefabFocusOperationResult FocusOnPrefabInstanceOwningEntityId(AZ::EntityId entityId) override;
+        PrefabFocusOperationResult OpenPrefabInstanceOwningEntityIdAndDescendants(AZ::EntityId entityId) override;
```

Comment:
This could likely actually be a PrefabIntegrationManager function, since it doesn't really belong here in the Focus handler... but we'll see.